### ISSUE TITLE
Figure out why emails are no longer going to the console when developing

### ISSUE THREAD

**Issue by tobiasmcnulty** [caktus/etlman#49]
A/C:

- [x] Update README above the `createsuperuser` step to include the message:

> When creating a super user, note that you must provide an email address, otherwise you will not be able to verify your account (via the link in the email printed to the console) nor login to the system.

**Comment by ronardcaktus** [comment]
I realized that my issue was that when I created my super user, I left the `email` field blank. This feature might be working fine, I'd like it if another dev tested this.
![Screen Shot 2022-08-04 at 2 08 51 PM](https://user-images.githubusercontent.com/87770895/182921215-9ae1deb9-f5ae-4fd5-888e-3cd5894c092c.png)

**Comment by wfzyx** [comment]
I think it may be related to the version bump we did on Django?

**Comment by wfzyx** [comment]
So apparently, this is intended behavior. I couldn't track the change that caused this behavior to change, either in etlman or allauth. But going by the docs and the unit tests on allauth, I'm pretty sure you are not supposed to create an user without an email, when email confirmation is required. From the docs:

```
ACCOUNT_EMAIL_REQUIRED (=False)
The user is required to hand over an e-mail address when signing up.
ACCOUNT_EMAIL_VERIFICATION (=”optional”)
Determines the e-mail verification method during signup – choose one of "mandatory", "optional", or "none".

Setting this to “mandatory” requires ACCOUNT_EMAIL_REQUIRED to be True
```

https://django-allauth.readthedocs.io/en/latest/configuration.html

Maybe this should be enforced on createsuperuser on django side ? (Ticket for 4.2.X?) 

@tobiasmcnulty FYI

**Comment by ronardcaktus** [comment]
Addressed in PR #117
